kubernetes/zenko/values.yaml

```diff
@@ -76,6 +76,11 @@ cloudserver:
     # accountName:
       # access:
       # secret:
+  # externalBackends:
+  #   aws_s3:
+  #     keepAlive: false
+  #   gcp:
+  #     keepAlive: true
```

Comment:
IMO, there is no need to put values in the Zenko chart when the defaults from the sub-chart don't need to be overwritten. It makes the values file more complicated than what it is.

Reply:
Yeah I understand. I was debating of a way of making them configurable easier but I can see it can be a downhill going this path.